livingatlas/pipelines/src/main/java/au/org/ala/pipelines/beam/IndexRecordPipeline.java

```diff
@@ -182,59 +188,75 @@ public static void run(ALASolrPipelineOptions options) throws Exception {
     if (options.getIncludeSensitiveData()) {
       alaSensitiveDataCollection =
           p.apply("Read sensitive data", alaSensitiveDataRecordTransform.read(pathFn))
-              .apply("Map sensitive to KV", alaSensitiveDataRecordTransform.toKv());
+              .apply("Map attribution to KV", alaSensitiveDataRecordTransform.toKv());
```

Comment:
This doesn't seem right. Possibly a global search/replace?

Reply:
yep. ill fix